protocol/0083-RFPR-on_chain_referral_program.md

```diff
@@ -46,8 +46,14 @@ Enabling or changing the terms of the on-chain referral program can be proposed
 - `benefit_tiers`: a list of dictionaries (with the below fields) defining the reward and discount factors from referrals
   - `minimum_running_notional_taker_volume`: the required [`referral_set_running_notional_taker_volume`](#referral-set-volumes) in quantum units for parties to access this tier
   - `minimum_epochs`: the required number of epochs a referee must have been in a referral set to access this tier
-  - `referral_reward_factor`: the proportion of the referee's taker fees to be rewarded to the referrer
-  - `referral_discount_factor`: the proportion of the referee's taker fees to be discounted
+  - `referral_reward_factors`: the proportions of the referee's taker fees to be rewarded to the referrer
+    - `maker_volume_reward_factor`: the factor to reward the referrer from the maker fee component of any fees paid
+    - `infrastructure_volume_reward_factor`: the factor to reward the referrer from the infrastructure fee component of any fees paid
+    - `lp_volume_reward_factor`: the factor to reward the referrer from the liquidity provider fee component of any fees paid
```

Comment:
Slightly prefer the parameter names chosen in `0029` as it makes it clear this reward is coming from the referral program, but have no strong preference other than being consistent (so this suggestion or update `0029` to match the names here).

```suggestion
    - `referral_maker_reward_factor`: the factor to reward the referrer from the maker fee component of any fees paid
    - `referral_infrastructure_reward_factor `: the factor to reward the referrer from the infrastructure fee component of any fees paid
    - `referral_liquidity_reward_factor`: the factor to reward the referrer from the liquidity provider fee component of any fees paid
```